packages/typespec-client-generator-core/src/lib.ts

```diff
@@ -13,6 +13,10 @@ const EmitterOptionsSchema: JSONSchemaType<SdkEmitterOptions> = {
     "examples-directory": { type: "string", nullable: true },
     "examples-dir": { type: "string", nullable: true },
     "emitter-name": { type: "string", nullable: true },
+    "license-name": { type: "string", nullable: true },
```

Comment:
would it be better to have a flag structure more like this?
```yaml
license?:
    name: str
    company: str
    link?: str
    description?: str
```

This is just my guess for what would be required in a license property and what wouldn't be

Reply:
Personally, I like the structured form too.

One thing to know, previously we are not able to use the command line to set nested property.
e.g. we can do `--option "@typespec/http-client-java.license-name=..."`, but not `--option "@typespec/http-client-java.license.name=..."`
I am not sure whether tsp supports it now.

Reply:
changed to form.